hyundai_kia_generic.dbc

```diff
@@ -1443,4 +1443,12 @@ BO_ 905 SCC14: 8 SCC
  SG_ JerkUpperLimit : 12|7@1+ (0.1,0) [0|12.7] "m/s^3"  ESC
  SG_ JerkLowerLimit : 19|7@1+ (0.1,0) [0|12.7] "m/s^3"  ESC
  SG_ SCCMode : 32|3@1+ (1,0) [0|7] ""  ESC
+ 
+BO_ 870 366_EMS: 8 EMS
+ SG_ EMS_Related : 23|16@0+ (1,0) [0|65535] "" XXX
```

Comment:
you sure it's big endian? everything else is little endian

Reply:
Unaware. I ID'd it and the plot matched x EMS11